html-relative-links/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home Page</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="">Home Page</a></li>
```

Comment:
In this case since it's in the same directory, you can just do `href="index.html"`